### ISSUE TITLE
Readonly status change on dictionary reload are not updated

### ISSUE THREAD

**Issue by user202729** [openstenoproject/plover#1399]
**Describe the bug**

title.

**To Reproduce**

1. Add a dictionary.
2. Set the dictionary file attribute to readonly (with external tool)
3. Refresh the dictionaries in Plover.
4. The dictionary is not shown as readonly. (error)
5. Add an entry to that dictionary.
6. The dictionary is written to normally, and the readonly flag disappears. (error)

**Expected behavior**

~

**Operating system**

 - OS: Some GNU/Linux distribution
 - Plover Version: https://github.com/openstenoproject/plover/commit/0501aa33a0504971d1ba7452979c54798f7c3986

**Comment by user202729** [comment]
Update: The status is updated if the file **content** is changed.

**Comment by benoit-pierre** [comment]
Yes, Plover relies on timestamps. So I'm tempted to close this as wont-fix.

**Comment by mkrnr** [comment]
Closing this ticket due to inactivity. Please comment if this is still an issue.
